script.js

```diff
@@ -1,3 +1,20 @@
 var Vue = new Vue({
-	el: 'body',
```

Comment:
I use mount to  `#app ` instead of  `body`.  
we know the way recommended on vue 2.
should split commit?